### PR TITLE
Adding bio for slynch0t

### LINKED ISSUE #52: Add bio for slynch0t
- [x] Create a branch for slynch0t
- [x] create a bio for slynch0t
- [x] Open a pull request
- [ ] Get and Receive feedback
- [ ] Merge Pull request

### PR DESCRIPTION
**Pull request by slynch0t**
This will fix #52

### CODE REVIEW

**Comment by vinothsg11** [comment]
Good job

File: slynch0t.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#slynch0t
+
+*Name: slynchot
```
Comment (vinothsg11) [inline slynch0t.md:3]: Good job

File: slynch0t.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#slynch0t
+
```
Comment (kennym52) [inline slynch0t.md:2]: What up homie!

Comment (slynch0t) [inline slynch0t.md:2]: Oh no, @kennym52 , you've found me!

**Pull request merged**